ecosystem/scenarios/test_requests.py

```
@@ -0,0 +1,212 @@
+
+NOTE: Currently fails due to a bug in cribo's handling of relative imports
+      like "from . import sessions" which get transformed incorrectly.
+      TODO: Remove tree_shake=False once the relative import bug is fixed.
+"""
+
+import sys
+from pathlib import Path
+from types import ModuleType
+from typing import TYPE_CHECKING, Type
+
+from .utils import run_cribo, format_bundle_size, load_bundled_module, ensure_test_directories, get_package_requirements
```

Pylint (3.3.7)

[convention] Line too long (120/100)

(C0301)